src/index.js.flow

```diff
@@ -419,6 +419,11 @@ export type DAOConfig<TEntity, TEntityMutator> = {
 };
 
 declare export class DAOResult<TEntity> {
+  constructor(
```

Comment:
with DAOResult and DaoResultSet, it will be   `|` in every place where we actually use the types, but with one type we hide it here